src/App.jsx

```diff
@@ -0,0 +1,48 @@
+/* eslint-disable react-refresh/only-export-components */
+import "./index.css";
+import Header from "./Components/Header";
+import NavAside from "./Components/NavAside";
+import Feed from "./Components/Feed";
+import Profile from "./Components/Profile";
+import SinglePost from "./Subcomponents/SinglePost";
+import { Routes, Route } from "react-router-dom";
+import { createContext, useEffect, useState } from "react";
+
+export const loggedInUser = createContext();
+
+function App() {
+  const [loggedUser, setLoggedUser] = useState();
+
+  useEffect(() => {
+    logUserIn();
+  }, []);
+
+  const logUserIn = async () => {
+    const data = await fetch(
+      "https://boolean-uk-api-server.fly.dev/MrStashy/contact/1"
```

Comment:
`https://boolean-uk-api-server.fly.dev/MrStashy/` put it into a const in its own file, maybe `constants.js` that you can import when needed

Reply:
`apiUrl + '/contact'`